src/deploy-cromwell-on-azure/Utility.cs

```diff
@@ -2,13 +2,68 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Security.Cryptography;
 using System.Text.RegularExpressions;
 
 namespace CromwellOnAzureDeployer
 {
     public static class Utility
     {
+        public static string GetFileContent(params string[] pathComponentsRelativeToAppBase)
+        {
+            var embeddedResourceName = $"deploy-cromwell-on-azure.{string.Join(".", pathComponentsRelativeToAppBase)}";
+            var embeddedResourceStream = typeof(Deployer).Assembly.GetManifestResourceStream(embeddedResourceName);
+
+            using var reader = new StreamReader(embeddedResourceStream);
+            return reader.ReadToEnd().Replace("\r\n", "\n");
+        }
+
+        public static string UpdateExistingSettingsFileContentV300(string existingSettingsFileContent)
```

Comment:
This return is there just for unit testing. OK for now, we should refactor this and move all the "patch" methods to separate class to reduce clutter in Deployer.cs. If you want to unit test, than I suggest adding a "proxy" for the VM and create create a mock VM in that case.